docs/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rollbacks.md

```diff
@@ -1,6 +1,26 @@
 ---
 title: Rollbacks
 ---
+## Additional Steps for Rollbacks with Rancher v2.6.4+
+
+In Rancher v2.6.4, the cluster-api module has been upgraded from v0.4.4 to v1.0.2 in which the apiVersion of CAPI CRDs are upgraded from `cluster.x-k8s.io/v1alpha4` to `cluster.x-k8s.io/v1beta1`. This has the effect of causing rollbacks from Rancher v2.6.4 to any previous version of Rancher v2.6.x to fail because the previous version the CRDs needed to roll back are no longer available in v1beta1.
+
+To avoid this, the Rancher resource cleanup scripts should be run **before** the restore or rollback is attempted. Specifically, two scripts have been created to assist you: one to clean up the cluster (`cleanup.sh`), and one to check for any Rancher-related resources in the cluster (`verify.sh`). Details on the cleanup script can be found in the [rancher/rancher-cleanup repo](https://github.com/rancher/rancher-cleanup).
```

Comment:
```suggestion
To avoid rollback failure, the Rancher resource cleanup scripts should be run **before** the restore operation or rollback is attempted. Specifically, two scripts have been created to assist you: 

* `verify.sh`:  Checks for any Rancher-related resources in the cluster. 
*  `cleanup.sh`: Cleans up the cluster.

Details on the cleanup script can be found in the [rancher/rancher-cleanup repo](https://github.com/rancher/rancher-cleanup).
```